### DISCUSSION TITLE
Help with MacOS commond error

### DISCUSSION THREAD

**Discussion by buildplan** [simulot/immich-go#581] [Q&A] [answered]
Hi, Apologies for a noob question, I am not good with CLI.

 I am using this command on Mac: 
`./immich-go -server=http://192.168.xx.xx:2283 -key=knIoMbY8nbiKXeIf7V0Kxxxxxxxxxx upload -create-stacks -create-albums -google-photos -dry-run takeout-*.zip`

I have immich-go and zip folders all in the same folder and terminal opened in that folder, I keep getting error like this:
_Error: unknown shorthand flag: 'c' in -create-stacks_

I have tried to remove `-create-stacks` flag, but the error just shifts to whatever command is after `upload`. Can anyone please point out something very obvious which I am doing wrong? Thanks.

**Comment by simulot** [comment]
Are you trying the stable release or the release candidate? The RC uses a different convention for the CLI

**Reply by buildplan** [reply]
Hi simulot, thank you very much for taking time to respond. I downloaded the latest release and now that you asked I checked its RC. Should I be using stable release, or is it OK to go with the latest RC? If it's OK, please advise on the syntax for that. Thank you again.

**Reply by simulot** [reply, marked as answer]
The RC version has been downloaded by 160 persons. I haven't received bad reports other than a "BAD REQUEST" on the stack operations. I'm trying to fix it  before making a release.

You can read an updated documentation here:
https://github.com/simulot/immich-go/tree/next

**Reply by buildplan** [reply]
Hi, thanks again for taking time to respond. I decided to go with the stable version because I could figure out the proper syntax for the options I wanted to use. It's totally on me. 
I am running this now, and it's been going for over 12 hours now. I had about 84k files, and I am about a quarter of the way through. Thanks for this tool, really appreciate it. 
![PXL_20241229_113839799](https://github.com/user-attachments/assets/7c06f92b-0dc9-42f2-99de-35ad91fbb28a)
